external/js/cothority/lib/protobuf/models.json

```diff
@@ -1 +1 @@
"id":1}}},"Questionnaire":{"fields":{"title":{"rule":"required","type":"string","id":1},"questions":{"rule":"repeated","type":"string","id":2},"replies":{"rule":"required","type":"sint32","id":3},"balance":{"rule":"required","type":"uint64","id":4},"reward":{"rule":"required","type":"uint64","id":5},"id":{"rule":"required","type":"bytes","id":6}}},"Reply":{"fields":{"sum":{"rule":"repeated","type":"sint32","id":1,"options":{"packed":true}},"users":{"rule":"repeated","type":"bytes","id":2}}},"RegisterQuestionnaire":{"fields":{"questionnaire":{"rule":"required","type":"Questionnaire","id":1}}},"ListQuestionnaires":{"fields":{"start":{"rule":"required","type":"sint32","id":1},"number":{"rule":"required","type":"sint32","id":2}}},"ListQuestionnairesReply":{"fields":{"questionnaires":{"rule":"repeated","type":"Questionnaire","id":1,"options":{"packed":false}}}},"AnswerQuestionnaire":{"fields":{"questid":{"rule":"required","type":"bytes","id":1},"replies":{"rule":"repeated","type":"sint32","id":2,"options":{"packed":true}},"account":{"rule":"required","type":"bytes","id":3}}},"TopupQuestionnaire":{"fields":{"questid":{"rule":"required","type":"bytes","id":1},"topup":{"rule":"required","type":"uint64","id":2}}},"Message":{"fields":{"subject":{"rule":"required","type":"string","id":1},"date":{"rule":"required","type":"uint64","id":2},"text":{"rule":"required","type":"string","id":3},"author":{"rule":"required","type":"bytes","id":4},"balance":{"rule":"required","type":"uint64","id":5},"reward":{"rule":"required","type":"uint64","id":6},"id":{"rule":"required","type":"bytes","id":7},"partyiid":{"rule":"required","type":"bytes","id":8}}},"SendMessage":{"fields":{"message":{"rule":"required","type":"Message","id":1}}},"ListMessages":{"fields":{"start":{"rule":"required","type":"sint32","id":1},"number":{"rule":"required","type":"sint32","id":2},"readerid":{"rule":"required","type":"bytes","id":3}}},"ListMessagesReply":{"fields":{"subjects":{"rule":"repeated","type":"string","id":1},"msgids":{"rule":"repeated","type":"bytes","id":2},"balances":{"rule":"repeated","type":"uint64","id":3,"options":{"packed":true}},"rewards":{"rule":"repeated","type":"uint64","id":4,"options":{"packed":true}},"partyiids":{"rule":"repeated","type":"bytes","id":5}}},"ReadMessage":{"fields":{"msgid":{"rule":"required","type":"bytes","id":1},"partyiid":{"rule":"required","type":"bytes","id":2},"reader":{"rule":"required","type":"bytes","id":3}}},"ReadMessageReply":{"fields":{"message":{"rule":"required","type":"Message","id":1},"rewarded":{"rule":"required","type":"bool","id":2}}},"TopupMessage":{"fields":{"msgid":{"rule":"required","type":"bytes","id":1},"amount":{"rule":"required","type":"uint64","id":2}}}}},"status":{"options":{"java_package":"ch.epfl.dedis.lib.proto","java_outer_classname":"StatusProto"},"nested":{"Request":{"fields":{}},"Response":{"fields":{"status":{"keyType":"string","type":"onet.Status","id":1},"serveridentity":{"type":"network.ServerIdentity","id":2}}}}}}}
```

Comment:
We need to warn Gaylor about this change, so he can adapt to it in cothority TS, which is under construction right now.